nf-core-contributors.yaml

```diff
@@ -132,3 +132,16 @@ contributors:
       contact_github: mfoll
       location: [45.7438643, 4.8735019]
       twitter: iarcwho
+      
+    - full_name: Lifebit Biotech
+      short_name: Lifebit
+      description: >
+      Intelligent, automated and reproducible omics analyses at scale. Zero onboarding. Zero config.
```

Comment:
Hi Phil! Everything good, but can you change this to something less commercially-like? e.g. just state something like "Lifebit provides automated and reproducible omics analyses as a platform."

Reply:
Yes, same here. A formal description without any commercial subtext would be great.